### ISSUE TITLE
AZURE_CLIENT_SECRET is not set. testExplorer.html will not work

### ISSUE THREAD

**Issue by dluc** [Azure/cosmos-explorer#548]
Could you provide an actionable error message? For instance:
* where can one set this variable?
* what should the variable contain?
Thanks

**Comment by southpolesteve** [comment]
@dluc Thanks for the feedback. The testExplorer config is a very recent addition, and I do need to add some more docs around it. If you are just trying to do local dev, most of the team uses the "Hosted" version which is what you'll find at https://cosmos.azure.com/. https://localhost:1234/hostedExplorer.html will get you the exact same thing locally. 

The other option is to use the Azure Portal as a wrapper which is performing the same function as testExplorer: https://ms.portal.azure.com/?dataExplorerSource=https%3A%2F%2Flocalhost%3A1234%2Fexplorer.html#home

**Comment by dluc** [comment]
@southpolesteve I discovered https://cosmos.azure.com yesterday thanks to your docs :-) Yesterday, there was a major AAD outage and Azure Portal was semi-broken, even after recovering CosmosDb Data Explorer was not working and returning a blank page.  https://cosmos.azure.com really helped in that situation. And since it's under "azure.com" I could trust it with credentials etc. 

However, I'd love running the explorer locally, when I tried the "npm" commands I got stuck, so I hope you'll find the time to  make it easier. (btw I work on MacOS, so I can't leverage CosmosDb emulator without a Windows VM).

Overall, I think it's a great idea opening these tools via GitHub, thanks!
